app/controllers/survey_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SurveyController < ApplicationController
+  def index
+      puts "displaying survey"
```

Comment:
I'm kind of a stickler for this, unless they're super necessary we should avoid having print debugging lines. If we do need these lines, we should look for a logging library to integrate.